functions/exercise19/scriptSpect.js

```diff
@@ -0,0 +1,30 @@
+/// <reference path="../../typings/globals/jasmine/index.d.ts" />
+/*globals returnElementsLargerThanNumber*/
+
```

Comment:
Please add some tests for arrays with:
1. all items larger
2. items not sorted (to show, that input array does not have to be sorted)
2. some items with negative values (like -5 and -15).